bessctl/commands.py

```diff
@@ -1781,18 +1781,26 @@ def get_total(arr):
             print_header(now[port].timestamp)
 
             for port in ports:
-                print_delta('%s/%s' % (port, drivers[port]),
-                            get_delta(last[port], now[port]))
+                print_delta(now[port].timestamp, '{}{}'.format(port, drivers[port]),
+                            get_delta(last[port], now[port]), csv_f)
 
             print_footer()
 
             if len(ports) > 1:
-                print_delta('Total', get_delta(
+                print_delta(now[port].timestamp, 'Total', get_delta(
                     get_total(list(last.values())),
-                    get_total(list(now.values()))))
+                    get_total(list(now.values()))), csv_f)
 
             for port in ports:
                 last[port] = now[port]
+
+    try:
+        csv_path = os.getenv('CSV', None)
+        with open(csv_path, 'w') if csv_path is not None else noop() as csv_f:
+            if csv_f is not None:
+                csv_f.write('{}\n'.format(','.join(
+                    ('Timestamp', 'Port', 'Mbps_In', 'Mpps_In', 'Dropped_In', 'Mbps_Out', 'Mpps_Out', 'Dropped_Out'))))
+            print_loop(csv_f)
```

Comment:
Tiny things again, The output to `file` and `stdout` are slightly different in format and header name. Do you make it on purpose?

Reply:
The difference in the headers is intentional, but only because the text sent to `stdout` is more tabular and doesn't map cleanly to a csv. The values as read from left-to-right is meant to be equivalent, though